lighthouse-core/lib/manifest-parser.js

```diff
@@ -151,7 +151,7 @@ function parseStartUrl(jsonInput, manifestUrl, documentUrl) {
     return {
       raw,
       value: documentUrl,
-      warning: 'ERROR: invalid start_url relative to ${manifestUrl}',
+      warning: `ERROR: invalid start_url relative to ${manifestUrl}`,
```

Comment:
lol, that's certainly one big problem with testing via `assert.ok(parsedUrl.warning)`...